### PR TITLE
Made explanations clearer.

### PR DESCRIPTION
**Pull request by teymour-aldridge**

### CODE REVIEW

File: src/concepts/html/README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ description: The procedural macro for generating HTML and SVG
 
 # Using html!
 
-The `html!` macro allows you to write declarative HTML and SVG for your components. If you've used React's JSX it will feel quite familiar.
+The `html!` macro allows you to write HTML and SVG code declaratively. It is similar to JSX (a Javascript extension which allows you to write HTML code inside of Javascript).
 
 **Important notes**
 
-1. There must be only one root node in an `html!` invocation
+1. The `html!` macro only accepts one root html node (you can counteract this by [using an empty tag](literals-and-expressions.md))
```
Comment (jstarry) [inline src/concepts/html/README.md]: Do you mean lists? `html! { <><Root1 /><Root 2 /></> }`

**Comment by teymour-aldridge** [comment]
Probably (I mean '<></>' – React’s fragments).

File: src/concepts/html/README.md (hunk 1 above)
Comment (jstarry) [reply src/concepts/html/README.md]: Yeah, we should mentioned lists and link there instead

**Comment by teymour-aldridge** [comment]
Ok

File: src/concepts/html/README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ description: The procedural macro for generating HTML and SVG
 
 # Using html!
 
-The `html!` macro allows you to write declarative HTML and SVG for your components. If you've used React's JSX it will feel quite familiar.
+The `html!` macro allows you to write HTML and SVG code declaratively. It is similar to JSX (a Javascript extension which allows you to write HTML code inside of Javascript).
 
 **Important notes**
 
-1. There must be only one root node in an `html!` invocation
+1. The `html!` macro only accepts one root html node (you can counteract this by [using fragments or iterators](literals-and-expressions.md))
```
Comment (jstarry) [inline src/concepts/html/README.md]: this is still linking to the wrong page

Comment (teymour-aldridge) [reply src/concepts/html/README.md]: Oops!

**Comment by jstarry** [comment]
No worries 😄 thanks!

**Pull request merged**